### ISSUE TITLE
Need to re-make after each install

### ISSUE THREAD

**Issue by mmohaveri** [xmm7360/xmm7360-pci#142]
I'm using the driver on ThinkPad X1 Yoga 4th gen on Ubuntu 20.04 with kernel 5.11.0-34-generic with Secure Boot off.

Everytime I restart my computer, `lte up` will result in the following error:
`insmod: ERROR: could not insert module xmm7360.ko: Invalid module format`

Then based on [this](https://github.com/xmm7360/xmm7360-pci/issues/37#issuecomment-620389940) comment I clean the build, rebuild and re-load and then `lte up` will result in a successful launch.

**Comment by tgxn** [comment]
Yep, this is a known issue: https://github.com/xmm7360/xmm7360-pci/issues/1#issuecomment-744562299

I believe it's got something to do with the way suspend/resume is handled.
